plugins/search/src/components/SearchResult/index.tsx

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 Spotify AB
+ * Copyright 2021 Spotify AB
```

Comment:
I just want to mention that these years are normally not necessary to be changed.